src/pymor/operators/interface.py

```diff
@@ -62,6 +62,9 @@ class Operator(ParametricObject):
         for all V, mu.
     """
 
+    # override NumPy binary operations and ufuncs
```

Comment:
Can you explain this a little bit further?

Reply:
> I now realize I didn't copy the third line from SciPy (`__array_ufunc__ = None`). This should also not let something like `np.linalg.norm(vector_array)` fail. There is info [here](https://numpy.org/doc/stable/reference/arrays.classes.html?highlight=__array_ufunc__#numpy.class.__array_ufunc__).

What do you mean by that? `np.linalg.norm(vector_array)` *should* fail. Shouldn't it?

Reply:
I meant to say, it shouldn't fail horribly. But, `np.linalg.norm` is actually not a ufunc (so it fails horribly either way), but something like `np.abs` is. This is what happens without `__array_ufunc__ = None`:

```python
>>> import numpy as np
>>> from pymor.vectorarrays.numpy import NumpyVectorSpace
>>> v = NumpyVectorSpace.from_numpy([1, 2])
>>> np.abs(v)
<stdin>:1: VisibleDeprecationWarning: Creating an ndarray from nested sequences exceeding the maximum number of dimensions of 32 is deprecated. If you mean to do this, you must specify 'dtype=object' when creating the ndarray.
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: bad operand type for abs(): 'NumpyVectorArray'
```

and this is what happens with `__array_ufunc__ = None`:

```python
>>> np.abs(v)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: operand 'NumpyVectorArray' does not support ufuncs (__array_ufunc__=None)
```

Reply:
Ok.